libraries/AP_RCTelemetry/AP_CRSF_Telem.h

```diff
@@ -33,6 +33,7 @@
 #include <AP_RCProtocol/AP_RCProtocol_CRSF.h>
 #include "AP_RCTelemetry.h"
 #include <AP_HAL/utility/sparse-endian.h>
+#include <AP_OSD/AP_OSD.h>
```

Comment:
Yeah I deliberately did not put this in here to avoid creating an include dependency on OSD from telemetry headers